pycode/memilio-epidata/memilio/epidata/getDIVIData.py

```diff
@@ -145,6 +144,9 @@ def get_divi_data(read_data=dd.defaultDict['read_data'],
 
     df = geoger.insert_names_of_states(df)
     df = geoger.insert_names_of_counties(df)
+    
+    # extract subframe of dates
+    df = mdfs.extract_subframe_based_on_dates(df, start_date, end_date)
```

Comment:
Why is this again? Because you have more dates (at beginning/end) due to the moving average? Please also see comment in extract_subframe_based_on_dates()

Reply:
Yes. Now it is only called once.